app/src/main/java/org/p2p/wallet/auth/interactor/AuthLogoutInteractor.kt

```diff
@@ -47,18 +46,4 @@ class AuthLogoutInteractor(
             pushNotificationsInteractor.deleteDeviceToken(publicKey)
         }
     }
-
-    fun clearAppData() {
```

Comment:
maybe we should stop the services at least?

Reply:
Oh, this method was duplicate of onUserLogout(), as you can see i clear everything, i just removed useless method